lte/gateway/c/oai/tasks/mme_app/mme_app_edns_emulation.c

```diff
@@ -57,6 +57,7 @@ int mme_app_edns_add_sgw_entry(bstring id, struct in_addr in_addr) {
 
       hashtable_rc_t rc =
           obj_hashtable_insert(g_e_dns_entries, cid, strlen(cid), data);
+      free(cid);
```

Comment:
what happens when data is null?

Reply:
That's an important catch,  moved `free` to line 62.

Reply:
Whoops! Thanks @ulaskozat for catching that!